### PR TITLE
Option for exclude default build deps

### PR DESCRIPTION
**Pull request by jean-helsinki**
for issue #133

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-from distutils.core import setup
+from setuptools import setup
```
Comment (dirk-thomas) [inline setup.py]: The switch to `setuptools` seems quite significant to me. I am not sure I can reasonably test this in all possible scenarios to ensure it doesn't break anything.

File: setup.py [hunk 2]
```diff
@@ -7,7 +7,7 @@
 setup(name='stdeb',
       # Keep version in sync with stdeb/__init__.py and install section
       # of README.rst.
-      version='0.8.5',
+      version='0.8.5.1',
```
Comment (dirk-thomas) [inline setup.py]: This change seems unrelated and should be reverted.

**Comment by dirk-thomas** [comment]
@jean-helsinki Friendly ping.

File: setup.py (hunk 2 above)
Comment (jean-helsinki) [reply setup.py]: Done

File: setup.py (hunk 1 above)
Comment (jean-helsinki) [reply setup.py]: Done reverting to distutils

File: stdeb/util.py [hunk 3]
```diff
@@ -879,6 +879,15 @@ def __init__(self,
         else:
             build_deps.append( 'debhelper (>= %s)'%DH_MIN_VERS )
 
+        try:
+            no_default_build_deps = \
+                cfg.get('DEFAULT', 'No-Default-Build-Deps')\
```
Comment (dirk-thomas) [inline stdeb/util.py:884]: Instead of accessing the config directly this should use `parse_val()` instead.

Please document the new configuration option in the README.

**Comment by dirk-thomas** [comment]
@jean-helsinki Friendly ping.

**Comment by dirk-thomas** [comment]
Closing due to no response from the author.
